cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255, null=True)
+    description = models.CharField(max_length=255, null=True)

```

Comment:
Similar to the `title` field, using `null=True` on `CharField` is not recommended. Consider using `blank=True` instead to allow empty values.